packages/vscode-extension/src/debug/depsChecker/vscodeTelemetry.ts

```diff
@@ -6,10 +6,9 @@ import { performance } from "perf_hooks";
 import { SystemError, UserError } from "@microsoft/teamsfx-api";
 import { TelemetryProperty } from "../../telemetry/extTelemetryEvents";
 import { ExtTelemetry } from "../../telemetry/extTelemetry";
-import { DepsCheckerEvent, TelemetryMessurement } from "./common";
-import { IDepsTelemetry } from "./checker";
+import { DepsTelemetry, DepsCheckerEvent, TelemetryMessurement } from "@microsoft/teamsfx-core";
 
-export class VSCodeTelemetry implements IDepsTelemetry {
+export class VscodeTelemetry implements DepsTelemetry {
```

Comment:
Better to keep naming consistency? I prefer `VSCodeTelemetry` which is consistent with `VSCodeDepsChecker` class in `vscodeChecker.ts`.

Reply:
Good idea, changed.